lib/webauthn/attestation_statement/packed.rb

```diff
@@ -89,8 +89,8 @@ def valid_signature?(authenticator_data, client_data_hash)
       end
 
       def attestation_type_and_trust_path
-        if raw_attestation_certificates&.any?
-          [WebAuthn::AttestationStatement::ATTESTATION_TYPE_BASIC_OR_ATTCA, attestation_certificate_chain]
+        if certificates&.any?
+          [WebAuthn::AttestationStatement::ATTESTATION_TYPE_BASIC_OR_ATTCA, certificates]
```

Comment:
How about using `attestation_trust_path` instead? I know it's just an alias for `certificates` but I think that, when possible, using the same terms as the spec makes it less confusing. It's also more consistent with `AttestationStatement::Base` and the other formats which already use the method.

Reply:
Agree. Better that way, thanks.